src/django/api/management/commands/remote.py

```diff
@@ -0,0 +1,164 @@
+                return response[key]
+
+    def get_task_def(self):
+        """
+        Get the ARN of the latest ECS task definition for the app CLI.
+        """
+        task_def_response = self.ecs_client.list_task_definitions(
+            familyPrefix=f'{self.env}AppCLI',
+            sort='DESC',
+            maxResults=1
+        )
+
+        return self.parse_response(task_def_response, 'taskDefinitionArns', 0)
+
+    def get_security_group(self):
+        """
+        Get the ID of the security group to use for the app CLI.
+        """
+        filters = [
+            {
+                'Name': 'tag:Name',
+                'Values': ['sgAppEcsService']
+            },
+            {
+                'Name': 'tag:Environment',
+                'Values': [self.env]
+            },
+            {
+                'Name': 'tag:Project',
+                'Values': ['OpenApparelRegistry']
+            }
+        ]
+
+        sg_response = self.ec2_client.describe_security_groups(
+            Filters=filters,
+        )
+
+        security_group = self.parse_response(sg_response, 'SecurityGroups', 0)
+        return security_group['GroupId']
+
+    def get_subnet(self):
+        """
+        Get a subnet ID to use for the app CLI.
+        """
+        filters = [
+            {
+                'Name': 'tag:Name',
+                'Values': ['PrivateSubnet']
+            },
+            {
+                'Name': 'tag:Environment',
+                'Values': [self.env]
+            },
+            {
+                'Name': 'tag:Project',
+                'Values': ['OpenApparelRegistry']
+            }
+        ]
+
+        subnet_response = self.ec2_client.describe_subnets(
+            Filters=filters,
+        )
+
+        subnet = self.parse_response(subnet_response, 'Subnets', 0)
+        return subnet['SubnetId']
+
+    def run_task(self, task_def_arn, security_group_id, subnet_id, cmd):
+        """
+        Run a task for a given task definition ARN using the given security
+        group and subnets, and return the task ID.
+        """
+        overrides = {
+            'containerOverrides': [
+                {
+                    'name': 'django',
+                    'command': cmd
+                }
+            ]
+        }
+
+        network_configuration = {
+            'awsvpcConfiguration': {
+                'subnets': [subnet_id],
+                'securityGroups': [security_group_id]
+            }
+        }
+
+        task_response = self.ecs_client.run_task(
+            cluster=f'ecs{self.env}Cluster',
+            taskDefinition=task_def_arn,
+            overrides=overrides,
+            networkConfiguration=network_configuration,
+            count=1,
+            launchType='FARGATE'
+        )
+
+        task = self.parse_response(task_response, 'tasks', 0)
+
+        # Task ARNS use the format arn:aws:ecs:<region>:<aws_account_id>:task/<id>
+        task_id = task['taskArn'].split('/')[1]
```

Comment:
This isn't super pretty, but [Boto doesn't return a task ID from `client.run_task`](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ecs.html#ECS.Client.run_task), so the choice was between substring parsing and an extra Boto call just to get the task ID.